test-case/check-performance.sh

```diff
@@ -62,11 +65,24 @@ do
         dlogi "Running (PCM: $pcm [$dev]<$type>) in background"
         if [ "$type" == "playback" ]; then
             aplay_opts -D "$dev" -c "$channel" -r "$rate" -f "$fmt" -d "$duration" /dev/zero -q &
+            aplay_num=$((aplay_num+1))
         else
             arecord_opts -D "$dev" -c "$channel" -r "$rate" -f "$fmt" -d "$duration" /dev/null -q &
+            arecord_num=$((arecord_num+1))
         fi
 done
 
+sleep 1 # waiting stable streaming of aplay/arecord
+dlogi "Number of aplay/arecord process started: $aplay_num, $arecord_num"
+
+real_aplay_num=$(ps --no-headers -C aplay | wc -l)
+real_arecord_num=$(ps --no-headers -C arecord | wc -l)
```

Comment:
You should use `wait $PIDs` instead of `ps`. It's more reliable and it passes error codes.